runelite-api/src/main/java/net/runelite/api/widgets/WidgetID.java

```diff
@@ -215,6 +216,7 @@
 
 	static class FixedViewport
 	{
+		static final int MULTI_INDICATOR = 20;
```

Comment:
Can you check to see if this is supposed to be just in Viewport? If not, please check what the widget is supposed to be in ResizableViewport (resizeable mode)?

Reply:
Certainly
Fixed: 
![2018-05-26_12-59-05](https://user-images.githubusercontent.com/13276766/40575414-f3fbe046-60e4-11e8-86d2-229ac72a43af.png)

Resizable
![java_2018-05-26_13-00-07](https://user-images.githubusercontent.com/13276766/40575415-f941f1c6-60e4-11e8-800e-f65e9fe84f8d.png)